modules/serverless-creating-trigger-admin-web-console.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-cluster-admin-eventing.adoc
+
+[id="serverless-creating-trigger-admin-web-console_{context}"]
+= Creating a trigger by using the Administrator perspective
+
+If you have cluster administrator permissions and have created a broker, you can create a trigger to connect your broker to a subscriber by using the Administrator perspective in the web console.
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Eventing are installed on your {product-title} cluster.
+* You have cluster administrator permissions for {product-title}.
+* You have created a broker.
+* You have created a Knative service to use as a subscriber.
+
+.Procedure
+
+. In the *Administrator* perspective of the {product-title} web console, navigate to *Serverless* -> *Eventing*.
+. In the *Broker* tab, select the image:../images/kebab.png[title="Kebab"] icon for the broker that you want to add a trigger to.
+. Click *Add Trigger* in the list.
+. In the *Add Trigger* dialogue box, select a *Subscriber* for the trigger. The subscriber is the Knative service that will receive events from the broker.
```

Comment:
Here also this is too limiting. The trigger can forward events to any Addressable object like kservice or Broker, or channel

Reply:
See: https://issues.redhat.com/browse/ODC-5525

Reply:
@cardil this ODC issue is still TO DO and hasn't been fixed yet, so currently AFAIK you can only sink to a service in the console. In fact, it won't even let you create a subscriber or trigger unless you've created a service first. I agree it's a terrible UX but we need to document what it actually does as opposed to what we'd like it to do.

Reply:
Thanks for the issue comment @abrennan89. That resolves those threads in here for me.